.pipelines/azure-pipeline-build.yml

```diff
@@ -336,7 +336,7 @@ jobs:
         oras attach $(LINUX_FULL_IMAGE_NAME) \
           --artifact-type 'application/vnd.cncf.notary.signature' \
           ./payload.json:application/cose \
-          -a "io.cncf.notary.x509chain.thumbprint#S256=[\"659AAA9C0E822B4B20A964AA0178BD9419A50530\"]"
+          -a "io.cncf.notary.x509chain.thumbprint#S256=[\"79E6A702361E1F60DAA84AEEC4CBF6F6420DE6BA\"]"
```

Comment:
hi soham, i just built without this change, and all built fine. So is this thumbprint change recent ?

Reply:
I think I missed to update the thumbprint when I changed the cert to prod cert under prod AKV. because while checking everything, I found this to be different from the prod cert. I think I missed to update this last time. I only did the service connection change.